hw03_gatewooda.py

```diff
@@ -0,0 +1,95 @@
+#################################################################################
+# Author: Austin Gatewood
+# Username:gatewooda
+#
+# Assignment:hw03 psychedelic turtles
+# Purpose:learn pull requests/ git in general
+# Google Doc Link:https://docs.google.com/document/d/1KOOO9_UM-Ka7Hg6IGKj9Xm-SKa6xDAZhze_jKE_fc78/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+import turtle
+import math
+
+def face(radius):    # super slow creation
+    """
+    Will create the face for the 3d glasses.
+    """
+    happy = turtle.Turtle()  # will create the face for the glasses
+    happy.hideturtle()
+    happy.speed(0)
+    happy.color("#087209")
+
+    # Move forward to the position where the face should start
+    happy.penup()
+    happy.goto(20, -radius)
+    happy.pendown()
+
+    # Draw a circle
+    for _ in range(360):
+        happy.forward(radius * 2 * math.pi / 360)     # this might be why
+        happy.left(1)
+
+
```

Comment:
Only 2 blank lines.

Reply:
I have changed where every function definition has two blank spaces between the previous function.